src/core/electric_utility.jl

```diff
@@ -559,22 +558,22 @@ function cambium_emissions_profile(; scenario::String,
                                     )
 
     url = "https://scenarioviewer.nrel.gov/api/get-levelized/" # Production 
-    project_uuid = "82460f06-548c-4954-b2d9-b84ba92d63e2" # Cambium 2022 
+    project_uuid = "0f92fe57-3365-428a-8fe8-0afc326b3b43" # Cambium 2023 
+    
 
     payload=Dict(
             "project_uuid" => project_uuid,
             "scenario" => scenario,
-            "location_type" => location_type,  # Nations, States, GEA Regions (Default: GEA Regions)
-            # "location" => "Colorado", # e.g., Contiguous United States, Colorado, Kansas, p33, p34 
+            "location_type" => location_type,  
             "latitude" => string(round(latitude, digits=3)),
             "longitude" => string(round(longitude, digits=3)), 
-            "start_year" => string(start_year), # biennial from 2022-2050 (data year covers nominal year and years proceeding; e.g., 2040 values cover time range starting in 2036)
+            "start_year" => string(start_year), # data year covers nominal year and 4 years proceeding; e.g., 2040 values cover time range starting in 2036
             "lifetime" => string(lifetime), # Integer 1 or greater (Default 25 yrs)
             "discount_rate" => "0.0", # Zero = simple average (a pwf with discount rate gets applied to projected CO2 costs, but not quantity.)
             "time_type" => "hourly", # hourly or annual
             "metric_col" => metric_col, # lrmer_co2e
             "smoothing_method" => "rolling", # rolling or none (only applicable to hourly queries). "rolling" best with TMY data; "none" best if 2012 weather data used.
-            "gwp" => "100yrAR6", # Global warming potential values. Default: "100yrAR6". Options: "100yrAR5", "20yrAR5", "100yrAR6", "20yrAR6" or a custom tuple [1,10.0,100] with GWP values for [CO2, CH4, N2O]
```

Comment:
Is this supposed to be reverted before merging?

Reply:
Oh great catch. I don't think this necessarily needs to be reverted because [1, 29.8, 273.0] should == using "100yrAR6". However, it would be good to find out if "100yrAR6" is still a viable option (currently, it errors). I'm still waiting to hear back from the Cambium team on this so let's keep this as an open TODO.